src/mmw/apps/bigcz/clients/cuahsi/search.py

```diff
@@ -228,6 +237,7 @@ def get_series_catalog_in_box(box, from_date, to_date, networkIDs):
     to_date = to_date or DATE_MAX
 
     result = make_request(client.service.GetSeriesCatalogForBox2,
```

Comment:
Can you speculate on the size of these responses so we have an idea of the impact to the cache storage we have available?

Reply:
```
vagrant@services:~$ du -sh *.txt
392K	GetSeriesCatalogForBox2.txt  # Cached for 5 minutes
144K	GetServicesInBox2.txt        # Cached for 1 week
```

Reply:
It should be noted that the original issue only asked to cache `GetServicesInBox2`. I'm also caching the other one to make interacting with it faster.